src/mongoose_client_api_messages.erl

```diff
@@ -15,11 +15,16 @@
 -export([maybe_integer/1]).
 -export([maybe_before_to_us/2]).
 
+-export([trails/0]).
+
 -include("mongoose.hrl").
 -include("jlib.hrl").
 -include("mongoose_rsm.hrl").
 -include_lib("exml/include/exml.hrl").
 
+trails() ->
+  mongooseim_swagger_docs:trails().
```

Comment:
Could you rename the `mongooseim_swagger_doc` module to `mongoose_client_api_messages_doc`?

Reply:
yes, but I was thinking - when we finished with architecture I will continue cover other REST APIs in module of `mongooseim_swagger_doc.erl`, actually this name is due to the fact that I intended to collect in one place and cover by Swagger documentation the all REST APIs. But maybe I shouldn't do this, since this module can be serve as a template for build Swagger documentation for other modules as of MongooseIM as for custom REST APIs

Reply:
Maybe create some sub folder eg `swagger_docs` then rename  `mongooseim_swagger_doc` to `mongoose_client_api_messages_doc` and move this module to new sub folder? then we can add new Swagger documentations in separate modules inside of  `swagger_docs` folder. What do you think about it?

Reply:
Done. Please, take a look if this approach is ok.